server/NewBank.java

```diff
@@ -87,4 +87,8 @@ public String getID(Customer c) {
 		}
 		return "";
 	}
+
+	public Customer getCustomer(String s){
```

Comment:
What is `s`? You should give your parameters meaningful names, in 99% of cases. 1% is for example when you have a for loop, and name a variable `i`